src/App.js

```diff
@@ -5,135 +5,166 @@
+import ChangeToken from './screens/ChangeToken';
+import TokenDetail from './screens/TokenDetail';
+import UnregisterToken from './screens/UnregisterToken';
+import ReceiveScreen from './screens/Receive';
+import Settings from './screens/Settings';
 
-const InitStack = createStackNavigator(
-  {
-    WelcomeScreen,
-    InitialScreen,
-    NewWordsScreen,
-    LoadWordsScreen,
-    BackupWords,
-    ChoosePinScreen,
-  },
-  {
-    initialRouteName: 'WelcomeScreen',
-    headerMode: 'none',
-  }
-);
+/**
+ * This Stack Navigator is exhibited when there is no wallet initialized on the local storage.
+ */
+const InitStack = () => {
+  const Stack = createStackNavigator();
+  return (
+    <Stack.Navigator
+      initialRouteName='Welcome'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='WelcomeScreen' component={WelcomeScreen} />
+      <Stack.Screen name='InitialScreen' component={InitialScreen} />
+      <Stack.Screen name='NewWordsScreen' component={NewWordsScreen} />
+      <Stack.Screen name='LoadWordsScreen' component={LoadWordsScreen} />
+      <Stack.Screen name='BackupWords' component={BackupWords} />
+      <Stack.Screen name='ChoosePinScreen' component={ChoosePinScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const DashboardStack = createStackNavigator(
-  {
-    Dashboard,
-    MainScreen,
-  },
-  {
-    initialRouteName: 'Dashboard',
-    headerMode: 'none',
-  },
-);
+const DashboardStack = () => {
+  const Stack = createStackNavigator();
+
+  return (
+    <Stack.Navigator
+      initialRouteName='Dashboard'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='Dashboard' component={Dashboard} />
+      <Stack.Screen name='MainScreen' component={MainScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const SendStack = createStackNavigator(
-  {
-    SendScanQRCode,
-    SendAddressInput,
-    SendAmountInput,
-    SendConfirmScreen,
-  },
-  {
-    initialRouteName: 'SendScanQRCode',
-    headerMode: 'none',
-  },
-);
+const SendStack = () => {
+  const Stack = createStackNavigator();
+
+  return (
+    <Stack.Navigator
+      initialRouteName='SendScanQRCode'
+      screenOptions={{
+        headerShown: false,
+      }}
+    >
+      <Stack.Screen name='SendScanQRCode' component={SendScanQRCode} />
+      <Stack.Screen name='SendAddressInput' component={SendAddressInput} />
+      <Stack.Screen name='SendAmountInput' component={SendAmountInput} />
+      <Stack.Screen name='SendConfirmScreen' component={SendConfirmScreen} />
+    </Stack.Navigator>
+  );
+};
 
-const CreateTokenStack = createStackNavigator(
-  {
-    CreateTokenDepositNotice,
-    CreateTokenName,
-    CreateTokenSymbol,
-    CreateTokenAmount,
-    CreateTokenConfirm,
-    CreateTokenDetail,
-  },
-  {
-    initialRouteName: 'CreateTokenDepositNotice',
-    headerMode: 'none',
-  }
-);
+const CreateTokenStack = () => {
```

Comment:
🚲 I think we should have a document string here like we see in InitStack.

Reply:
Done on [ae88d09](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/286/commits/ae88d09c6cdb386a0025367f7f7a6713205f4e72)